hack/common.sh

```diff
@@ -5,9 +5,9 @@
 
 GO111MODULE="on"
 GOPROXY="https://proxy.golang.org"
-GOROOT="${GOROOT:-go env GOROOT}"
+GOROOT="${GOROOT:-$(go env GOROOT)}"
```

Comment:
from the failure in the CI it looks like we don't have go in the openshift CI. We also need to figure this out.

Reply:
It is a chicken-and-egg problem:  you first need to be able to build ocs-operator over ARM64 before you may test it via CI.